ibis/expr/datatypes/value.py

```diff
@@ -267,9 +267,14 @@ def infer_shapely_multipolygon(value) -> dt.MultiPolygon:
 
 def normalize(typ, value):
     """Ensure that the Python type underlying a literal resolves to a single type."""
-    if value is None:
+    import pandas as pd
+
+    if value is None or isinstance(value, type(pd.NA)):
```

Comment:
could this use the same `pd.isna()` check that is used in the test?

Reply:
I don't think so, because I'm intentionally not including `NaN` in this check.